.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
This will run daily, at 5am. Once there are 5 update PRs in play, no more will be created until one is merged or closed.

Reply:
5 update PRs created by dependabot or in total?

Is it possible to have all the updates as one PR?

Reply:
5 update PRs created by dependabot.

Looking at their docs, it doesn't seem like they can batch the updates - which is a shame, as I know that other similar things, like Renovate, have that ability.